openfisca_france_fiscalite_miniere/tests/test_redevances.yaml

```
@@ -355,10 +355,10 @@
   output:
     redevance_communale_des_mines_antimoine:
       2021: 0
-      2022: 13.10 * 100 * (1/2)
+      2022: 13.80 * 100 * (1/2)
     redevance_departementale_des_mines_antimoine:
       2021: 0
-      2022: 3 * 100 * (1/2)
+      2022: 3.20 * 100 * (1/2)
```

> @MichaelBitard Les tests ont été effectivement pris avec un biais : l'application de la loi N en N+1 sur les données N (ce qu'on a fait jusqu'ici). D'où le fait qu'un taux 2021 ait été testé en 2022. 
Ce constat fait, nous pourrions mettre à jour pour tester sur la date d'entrée en vigueur des paramètres de la loi (3.20 en 2022 a priori ici) et garder l'historique supprimé ici en décalant vers le passé. Par exemple : 
```suggestion
      2020: 0
      2021: 3 * 100 * (1/2)
      2022: 3.20 * 100 * (1/2)
```

---

Ça me parait bien !